lab-kyle/test/hello-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Good job, the only suggestion I have here is to change the const hello -> const greet, it won't change the functionality at all but makes the variable name more meaningful, semantically.  Doing so, generally speaking, also makes it easier to trace the origin of module functions in your code (greet module => const greet, hello module => const hello, etc...).